src/Compilers/CSharp/Portable/FlowAnalysis/NullableWalker.cs

```diff
@@ -4868,10 +4959,12 @@ void trackConvertedValue(FieldSymbol targetField, Conversion conversion, FieldSy
                         if (AreNullableAndUnderlyingTypes(targetField.Type, valueField.Type, out _))
                         {
                             int targetFieldSlot = GetOrCreateSlot(targetField, slot);
+                            ValidateLastSlot();
```

Comment:
>ValidateLastSlot [](start = 28, length = 16)

Consider adding a `new int GetOrCreateSlot()` method in `NullableWalker` that simply delegates to `base.GetOrCreateSlot()` and calls `ValidateLastSlot()`.